core/trino-main/src/main/java/io/trino/operator/scalar/StringFunctions.java

```diff
@@ -968,4 +969,67 @@ public static Slice soundex(@SqlType(StandardTypes.VARCHAR) Slice slice)
         }
     }
+
+    @Description("Returns Fuzzy Match Simple Ratio")
+    @ScalarFunction("fuzzy_match_simple_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchSimpleRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.ratio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Partial Ratio")
+    @ScalarFunction("fuzzy_match_partial_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchPartialRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.partialRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Weighted Ratio")
+    @ScalarFunction("fuzzy_match_weighted_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchWeightedRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.weightedRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Token Sort Ratio")
+    @ScalarFunction("fuzzy_match_token_sort_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchTokenSortRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.tokenSortRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Token Sort Partial Ratio")
+    @ScalarFunction("fuzzy_match_token_sort_partial_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchTokenSortPartialRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.tokenSortPartialRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Token Set Ratio")
+    @ScalarFunction("fuzzy_match_token_set_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchTokenSetRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.tokenSetRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
+    }
+
+    @Description("Returns Fuzzy Match Token Set Partial Ratio")
+    @ScalarFunction("fuzzy_match_token_set_partial_ratio")
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long fuzzyMatchTokenSetPartialRatio(@SqlType("varchar(x)") Slice stringX, @SqlType("varchar(y)") Slice stringY)
+    {
+        return FuzzySearch.tokenSetPartialRatio(stringX.toStringUtf8(), stringY.toStringUtf8());
```

Comment:
Please consider using static imports for the static methods of the `FuzzySearch` class.